splight_lib/client/hub/abstract.py

```diff
@@ -21,7 +21,7 @@ def _get(
         pass
 
     @abstractmethod
-    def upload(self, data: Dict, files: Dict) -> Tuple:
+    def upload(self, id: str, file_path: str, type: str) -> BaseModel:
```

Comment:
```suggestion
    def upload(self, id: str, file_path: str, type: str):
```
Change to follow the implementation. Also, in the clients of the library we are not using the pydantic BaseModel, only python built-in types